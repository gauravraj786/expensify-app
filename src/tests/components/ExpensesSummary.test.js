import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpenseSummary';

test('should correctly render ExpenseSummary with 1 expense',() => {
    const wrapper = shallow(<ExpenseSummary expneseCount={1} expenseTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpenseSummary with multiple expenses',() => {
    const wrapper = shallow(<ExpenseSummary expneseCount={23} expenseTotal={2359765987} />);
    expect(wrapper).toMatchSnapshot();
});