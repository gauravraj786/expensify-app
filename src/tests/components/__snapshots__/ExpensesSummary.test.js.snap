// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpenseSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totaling 
    $2.35
    .
  </h1>
</div>
`;

exports[`should correctly render ExpenseSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totaling 
    $23,597,659.87
    .
  </h1>
</div>
`;
